packages/next-frontend/styles/components/hype-registration-button.module.css

```diff
@@ -28,3 +28,39 @@
 .balance:active {
 	box-shadow: inset 2px 1px 5px 1px rgba(255, 255, 255, 0.5);
 }
+.hypeMenu {
+	/* margin: 0;
+	position: absolute;
+	background-color: black;
+	width: 100%; */
+
+	position: absolute;
+	top: 70px;
+	/* left: -100%;  */
+	/* min-height: 60px; */
+	/* opacity: 0; */
+	/* display: flex; */
+	/* width: 100%; */
+	padding: 0rem 3rem;
+	justify-content: space-between;
+	align-items: center;
+	z-index: 9;
+	background: linear-gradient(180deg, rgba(63, 76, 80, 1),rgba(60, 45, 63, 1), rgba(47, 36, 39, 1));
```

Comment:
This is a small detail, so not super important, but personally I think using var(--grey-gradient) here would look a little cleaner and separate it from the main header